knative-operator/Gopkg.toml

```diff
@@ -45,7 +45,7 @@ required = [
 
 [[constraint]]
   name = "knative.dev/serving-operator"
-  branch = "master"
+  version = "v0.12.2"
```

Comment:
Should we vendor from our fork at `openshift-knative/serving-operator` for good measure? We could vendor the `release-0.12` branch to keep everything in sync.

Reply:
I'd rather not until we have a compelling reason to. I'm irrationally nervous about depending on a branch instead of a version.

Reply:
We should still vendor from our fork though.